example-documents/README.md

```diff
@@ -1,3 +1,50 @@
-this folder contains example documents 
-from past teams that we consider are of
-high quality. 
\ No newline at end of file
+This folder contains example documents from past teams that we consider are of high quality. 
+Use them as a rough guideline, not as a template.
+
+As a general note: try to keep your documents **consise**. That is: short but descriptive.
+The sad truth of the world is that people do not like to read large documents, and if your document is not being read you are wasting valuable development time ;).
+
+## Requirements Document
+The requirements document should be self contained and should  also make sense to somebody who is not part of the team. Thus it should have a high-level introduction that explains the importance and the scope of the project. But the most important role of this document, is that it will be the evolving shared understanding between you and the customer. Thus, the more details it contains the more can you be sure that you are on the same page. And this is why we ask you to update it after every meeting with the customer. 
+
+For the requirements, it is important that you note **why** the requirement should be implemented. As the reason will give context to the person implementing it. Have a look at the [Hestia](Hestia%20-%20Requirements%20Document%20Iteration%207.pdf) document if you want an example of how to do this using **User Stories**: _as an X I would like to Y so I can Z_. This will also make it easier for you to transfer your requirements document into a product backlog tool such as [JIRA](https://www.atlassian.com/software/jira) or the [Github Project Board](https://help.github.com/articles/about-project-boards/) (and vice-versa).
+
+## Example structure:
+- **A nice cover**:
+...will make the us happy. Add the project title, the client, the team names, TA name and the document version.
+- **Brief introduction**:
+A high-level introduction that explains the importance and the scope of the project. Explanation of the document structure.
+- **Critical Functional Requirements**:
+Requirements that your client would be very angry about if they are missing in the final product.
+- **Important Functional Requirements**:
+Requirements that would disappoint your client if they were missing in the final product.
+- **Useful Functional Requirements**:
+Requirements that your client would be happy about if they were implemented, but it's OK if they are not.
+- **Non-Functional Requirements**:
+All requirements that describe a certain _quality_ of the product, such as "Intuitive design" or "Android support". Be sure to properly understand the distinction between functional and non-functional (_a large part of students initially lose points due to this mistake!_).
+- **Won't Do**:
+Features you will not implement. Noting what you _will not_ do is just as important as noting what you _will_ do.
```

Comment:
A team might decide during a meeting that they will not support iOS _ever_ (not really a useful decision in my opinion, but they might do it), but their client might forget this choice and could demand this feature during the final iteration. Angry clients and sad teams ensue if it is not documented :-).